Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -955,13 +976,21 @@
     return false;
 }
 
-bool UnifiedPDFPlugin::handleEditingCommand(StringView commandName)
+bool UnifiedPDFPlugin::handleEditingCommand(const String& commandName, const String& argument)
 {
+    if (!m_frame || !m_frame->coreLocalFrame())
+        return false;
+    if (commandName == "ScrollPageBackward"_s || commandName == "ScrollPageForward"_s) {
+        m_frame->coreLocalFrame()->checkedEditor()->command(commandName).execute(argument);
+        return true;
+    }
     return false;
 }
 
-bool UnifiedPDFPlugin::isEditingCommandEnabled(StringView commandName)
+bool UnifiedPDFPlugin::isEditingCommandEnabled(const String& commandName)
 {
+    if (commandName == "ScrollPageBackward"_s || commandName == "ScrollPageForward"_s)
+        return true;
     return false;
```

Comment:
Nit:
```suggestion
    return commandName == "ScrollPageBackward"_s || commandName == "ScrollPageForward"_s;
```

Reply:
If I didn't know for sure that we were going to add more cases here, sure.